packaging/fedora/snapd.spec

```diff
@@ -430,12 +439,16 @@ popd
 
 # Install all systemd units
 pushd ./data/systemd
-%make_install SYSTEMDSYSTEMUNITDIR="%{_unitdir}"
+%make_install SYSTEMDSYSTEMUNITDIR="%{_unitdir}" BINDIR="%{_bindir}" LIBEXECDIR="%{_libexecdir}"
 # Remove snappy core specific units
 rm -fv %{buildroot}%{_unitdir}/snapd.system-shutdown.service
 rm -fv %{buildroot}%{_unitdir}/snap-repair.*
+rm -fv %{buildroot}%{_unitdir}/snapd.core-fixup.*
 popd
 
+# Rename snapd.core-fixup.sh
```

Comment:
Why is this renamed? The unit file referes to it with the `.sh` suffix. Having said this, this unit is only relevant on core systems so its fine to just not package it at all on a fedora/opensuse system.